LICENSE-eclipse.txt

```diff
@@ -0,0 +1,195 @@
+Clojure is licensed by the Eclipse Public License, below.
```

Comment:
we should definitely create a directory licenses to stop "polluting" root dir ...